spk/ffsync/src/wizard/install_uifile.sh

```diff
@@ -75,6 +39,7 @@ PAGE_FFSYNC_SETUP=$(/bin/cat<<EOF
         "subitems": [{
             "key": "wizard_ffsync_public_url",
             "desc": "Public URL",
+            "emptyText": "http://hostname.domain:8132",
```

Comment:
why not:
```json
    "defaultValue": "http://hostname.domain:8132/token/1.0/sync/1.5",
```
only "defaultValue" can be edited, "emptyText" is meant as prompt and will go away when the field is clicked.

isn't the `/token/1.0/sync/1.5` part required?

Reply:
No, the `/token/1.0/sync/1.5` part is not required for the config file. But it is required for the client setup.

Reply:
I want to use "emptyText" and not "defaultValue" because the example cannot work as-is and we don't want the user being able to click next without explicitly considering their correct value.

Reply:
So the missleading part is in the text above:
http://public-url:8132/token/1.0/sync/1.5

My understanding was, that you have to enter the "client visible URL" as written in the description.